docs/api.rst

```diff
@@ -118,17 +118,7 @@ Airflow webserver is publicly accessible, and you should probably use the deny a
     [api]
     auth_backend = airflow.api.auth.backend.deny_all
 
-Two "real" methods for authentication are currently supported for the API.
-
-To enabled Password authentication, set the following in the configuration:
-
-.. code-block:: bash
-
-    [api]
-    auth_backend = airflow.contrib.auth.backends.password_auth
-
-It's usage is similar to the Password Authentication used for the Web interface.
-
+Kerberos authentication is supported for the API as well.
 To enable Kerberos authentication, set the following in the configuration:
```

Comment:
Oh. Hmmmm! Does this change mean that the only auth options on the API is Kerberos, Allow All, or Deny All?

Reply:
Yes, that’s my understanding. Unless we keep models.User only for this contrib.auth.backends.password_auth (which is not making sense to me).

The same situation for webserver authentication. Quite a few auth options are no longer applicable.

Let’s check these changes carefully and ensures no wrong deletion is done. On the other hand, we may want to start adding more doc/examples for the FAB UI, like how OAuth or LDAP auth can be done under FAB UI.

Reply:
Or, instead of removing contrib.auth.backends.password_auth, we can refactor it to make it work with the “ab_user” table (instead of “users” table from models.User) which is by FAB.